app/views/admin/admin_users/_user_history.html.erb

```diff
@@ -13,17 +13,39 @@
       <% unless @log_items.blank?
         @log_items.each do |item| %>
         <tr>
-          <td><%= item.created_at %></td>
+          <td><%= item.created_at %> ( <abbr class="timeago" title="<%= item.created_at %>"><%= item.created_at.iso8601  %></abbr> )</td>
```

Comment:
There shouldn't be a space between the parentheses as the `<abbr>` tags. It should be `(<abbr class="timeago" title="<%= item.created_at %>"><%= item.created_at.iso8601 %></abbr>)`. There's also an extra space in `<%= item.created_at.iso8601  %>` that should be removed.